### PR TITLE
Heal Flavour!

### PR DESCRIPTION
**Pull request by Stagrawl**
First Step of Many!

## About The Pull Request

This flavours the T0 heal spell available to all of the Nine + Eora.

Just in case, I've included options for the other gods and Matthios to futureproof things. 

Zizo + Gragarr should realistically never get this spell, but hey! Just in case.

I think this is probably a good thing to do if we're establishing that all the Nine share one spell, as it keeps their specific usages distinguished.

## Why It's Good For The Game

This will make it clear who is healing you, adds flavour is coming from, etc, etc. I think this is just a good expansion for immersion's sake and will be far more useful when we have acolytes of other gods roaming around.

As for the greater heal spell, I'm not touching this for now as I do one day hope to make this Pestra exclusive.

### CODE REVIEW

**Comment by BlueMemesauce** [comment]
uh why are some of them randomly strings

File: code/modules/spells/roguetown/cleric.dm [hunk 1]
```diff
@@ -93,7 +93,37 @@
 			target.Paralyze(30)
 			target.fire_act(1,5)
 			return TRUE
-		target.visible_message("<span class='info'>A wreath of gentle light passes over [target]!</span>", "<span class='notice'>I'm bathed in holy light!</span>")
+		switch(user.patron.type)
+			if(/datum/patron/old_god)
+				target.visible_message("<span class='info'>A strange stirring feeling pours from [target]!</span>", "<span class='notice'>Sentimental thoughts drive away my pains!</span>")
+			if("/datum/patron/divine/astrata")
```
Comment (JTGSZ) [inline code/modules/spells/roguetown/cleric.dm:99]: they are correct about you just randomly swapping to strings in the condition checks in the switch statement

Comment (ChungusGamer666) [inline code/modules/spells/roguetown/cleric.dm:99]: string moment

File: code/modules/spells/roguetown/cleric.dm [hunk 2]
```diff
@@ -93,7 +93,37 @@
...
+			if(/datum/patron/inhumen/eora)
+				target.visible_message("<span class='info'>A lingering remanence of love diffuses through [target]!</span>", "<span class='notice'>My heart feels full, and my pains less severe!</span>")
+			if(/datum/patron/inhumen/zizo)
+				target.visible_message("<span class='info'>Vital energies are sapped towards [target]!</span>", "<span class='notice'>The life around me pales as I am restored!</span>")
+			if(/datum/patron/inhumen/graggar)
+				target.visible_message("<span class='info'>Foul fumes billow outward as [target] is restored!</span>", "<span class='notice'>A noxious scent burns my nostrils, but I feel better!</span>")
+			if(/datum/patron/inhumen/matthios )
+				target.visible_message("<span class='info'>A wreath of strange light passes over [target]!</span>", "<span class='notice'>I'm bathed in strange holy light?</span>")
+			if("/datum/patron/godless")
+				target.visible_message("<span class='info'>Without any particular cause or reason, [target] is healed!</span>", "<span class='notice'>My wounds close without cause.</span>")
```
Comment (ChungusGamer666) [inline code/modules/spells/roguetown/cleric.dm:126]: else statement needed

**Review by ChungusGamer666 — CHANGES_REQUESTED** [review]
strings everywhere wtf

**Pull request merged**